x-pack/plugins/security_solution/public/resolver/data_access_layer/mocks/one_ancestor_two_children.ts

```diff
@@ -40,11 +43,26 @@ interface Metadata {
 /**
  * A simple mock dataAccessLayer possible that returns a tree with 0 ancestors and 2 direct children. 1 related event is returned. The parameter to `entities` is ignored.
  */
-export function oneAncestorTwoChildren(): { dataAccessLayer: DataAccessLayer; metadata: Metadata } {
+export function oneAncestorTwoChildren(
+  {
+    withRelatedEvents,
+  }: { withRelatedEvents: Parameters<typeof withRelatedEventsOnOrigin>[1] | null } = {
```

Comment:
❔ Change back to It<string,string>

Reply:
https://github.com/elastic/kibana/pull/74411/commits/a48a10a6244e3ea35eda3de095fddb427e645c91#diff-7ac6cbbaa178435982e90bea4352bdf2R47